specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json

```diff
@@ -920,7 +928,11 @@
       "description": "Definition of the credential properties"
     },
     "Credential": {
-      "x-ms-azure-resource": true,
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
```

Comment:
On the other hand this changes might be breaking changes for some SDKs. WDYT about adding these improvements into a new file that is only used by the API version introduced in this PR?

---
In reply to: [177918048](https://github.com/Azure/azure-rest-api-specs/pull/2690#discussion_r177918048) [](ancestors = 177918048,177918018)

Reply:
Yes. But but I didn't want to change the meaning behind it. This can go to the discussion why have `ProxyResource` definition at all if all the resources can be either `TrackedResource` or just `Resource`. Having `ProxyResource` makes the spec clearer and explicit.

Reply:
What about the no longer needed properties?